docs/samples/specification/multiapi/generated/azure/multiapi/sample/v3/aio/operations/_operation_group_two_operations.py

```diff
@@ -68,15 +68,16 @@ async def test_four(  # pylint: disable=inconsistent-return-statements
         self,
         input: Optional[IO] = None,
         *,
-        content_type: Optional[str] = None,
+        content_type: str,
         **kwargs: Any
     ) -> None:
         """TestFour should be in OperationGroupTwoOperations.
 
         :param input: Input parameter. Default value is None.
         :type input: IO
         :keyword content_type: Body Parameter content-type. Content type parameter for binary body.
-         Default value is None.
+         Known values are: 'application/json', 'application/pdf', 'image/jpeg', 'image/png',
+         'image/tiff'. Required.
```

Comment:
Hi @iscai-msft I put `known values:` in both legacy and DPG, but it will not raise TypeError so that we could hint customers without break. What do you think about that?

Reply:
I think legacy should still have the same method signature, so legacy shouldn't change at all. It's technically a typing breaking change, and I feel like it's safer to leave legacy as it is